code/datums/emotes.dm

```diff
@@ -137,6 +131,18 @@
 			if(viewer.is_blind() && !viewer.can_hear())
 				to_chat(viewer, msg)
 
+	// SKYRAT EDIT -- BEGIN -- ADDITION -- AI QOL - RELAY EMOTES OVER HOLOPADS
+	var/obj/effect/overlay/holo_pad_hologram/h = GLOB.hologram_impersonators[user]
+	if(h)
+		if(emote_type & (EMOTE_AUDIBLE | EMOTE_VISIBLE))
+			h.audible_message(msg, deaf_message = "<span class='emote'>You see how <b>[user]</b> [msg]</span>", audible_message_flags = EMOTE_MESSAGE)
+		else if(emote_type & EMOTE_VISIBLE)
+			h.visible_message(msg, visible_message_flags = EMOTE_MESSAGE)
+		if(emote_type & EMOTE_IMPORTANT)
+			for(var/mob/living/viewer in viewers(world.view, h))
+				if(viewer.is_blind() && !viewer.can_hear())
+					to_chat(viewer, msg)
+	// SKYRAT EDIT -- END
```

Comment:
Single-letter variables are bad, give `h` a better name.

Also use `span_emote()` instead of baking the span classes in the string.